.scripts/update-expression-deps.rs

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env -S cargo +nightly -Zscript
```

Comment:
explain when/how to use this script in readme?

Reply:
[e1b432a](https://github.com/geo-engine/geoengine/pull/1011/commits/e1b432aa23dec9009e5fbe8eb05ca1abd58a190d)

Reply:
it's still not clear to me when I have to run this script. Should it be done everytime we update dependencies? Then maybe add it as a checklist item in the trello card.

Or can it also occur randomly if a dependency does something fishy like breaking semver?

If so, maybe we could somehow extend the CI to tell you to run this script? Would that make sense?

Reply:
https://github.com/geo-engine/geoengine/pull/1014